lib/table.py

```diff
@@ -110,7 +112,11 @@ def combine_on_delimiter(self, line_as_list):
         """Combine a list of values into a line of csv data"""
         dialect = csv.excel
         dialect.escapechar = "\\"
-        writer_file =  io.BytesIO()
+        if sys.version_info >= (3, 0):
+            writer_file =  io.StringIO()
+        else:
+            writer_file =  io.BytesIO()
```

Comment:
@goelakash you're totally right. it seems strange that it works this way, but clearly it does. Thanks for the explanation.